### DISCUSSION TITLE
Disabled buttons are too bright in dark mode

### DISCUSSION THREAD

**Discussion by fwcd** [community/community#113383] [Accessibility]
### Select Topic Area

Product Feedback

### Body

In dark mode, disabled buttons (e.g. in the pull request UI) have become distractingly bright in the most recent UI update:

![Screenshot 2024-03-20 at 01 25 03](https://github.com/community/community/assets/30873659/b4a54837-6ffa-467b-9c84-89303732814e)

I suspect the issue to be that they use the same color as light mode, where it looks fine:

![Screenshot 2024-03-20 at 01 25 11](https://github.com/community/community/assets/30873659/9b9afce9-0068-48e9-9b80-812c2eed3233)

It would be nice if this could be fixed, e.g. by making disabled buttons darker than normal buttons when in dark mode.

**Comment by jvmhonorato** [comment]
You probably have the "Dark Reader" extension activated in your browser. If you want dark mode, disable it and activate conventional dark mode in your browser settings.

**Comment by cehfisher** [comment]
Hi @fwcd  - thank you for providing this feedback! We wanted to follow up on this reported bug. We were unable to replicate this issue in our review. Is this an issue you can replicate currently? If the original issue is resolved, could you please close this out? If it is not resolved, could you supply more detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
